api_tests/src/follow.spec.ts

```diff
@@ -29,7 +28,7 @@ test("Follow federated community", async () => {
   // Make sure the follow response went through
   expect(betaCommunity?.community.local).toBe(false);
   expect(betaCommunity?.community.name).toBe("main");
-  expect(betaCommunity?.subscribed).toBe(SubscribedType.Subscribed);
+  expect(betaCommunity?.subscribed).toBe("Subscribed");
```

Comment:
Enums are all strings now?

Reply:
They were enum strings before (otherwise rust wouldn't have been able to deserialize them), but this library doesn't generate enums. Overall it seems pointless to create / duplicate the enum strings, since the strings are type-checked (IE only a few strings like "Subscribed" will work there).